konami/identifiers.md

```diff
@@ -96,7 +97,7 @@ The following variables are used across the different identifiers and can denote
 
 * Source: Game data
 * Format: `$e`
-* Example: `201807300200`
+* Example: `2018073002`
```

Comment:
This is wrong, even if older games would omit it, the example should cover the common case.

Reply:
No, you accidentally added a extra 00

Reply:
Nevermind, I'm stupid.